umbraco/Umbraco-CMS
# V8: Never user cculture when sorting the content tree

### Prerequisites


### Description

For some reason, `cculture` is used (if present) when sorting content nodes. That makes no sense, since the content tree is always shown in `mculture`:

![image](https://user-images.githubusercontent.com/7405322/55280215-a395ae80-5322-11e9-9869-1d8f09dd62c2.png)

This PR ensures that we only use `mculture` for sorting content.

#### Testing this PR

1. Edit a culture variant piece of content.
2. Change the editor to another language from the language dropdown (thus adding `cculture` to the querystring).
3. Open the sort dialog for any content node.
4. Verify that the node names in the sort dialog are using the default culture (`mculture`) and not the culture selected in the editor (`cculture`).
